service/src/test/java/uk/gov/hmcts/reform/fpl/service/config/NotificationContentProviderTestConfig.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.service.config;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.fpl.service.DateFormatterService;
+import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+
+@Configuration
+public class NotificationContentProviderTestConfig {
+    @Bean
+    @ConditionalOnMissingBean
+    public DateFormatterService dateFormatterService() {
+        return new DateFormatterService();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public HearingBookingService hearingBookingService() {
+        return new HearingBookingService();
+    }
```

Comment:
Still saying you don't need to do this. If you add `HearingBookingService.class` and `DateFormatterService.class` to the `@ SpringBootTest/@ContextConfiguration` it will do the same thing, just tested it on the `NoticeOfPlacementOrderUploadedEventHandlerTest`

Reply:
Also if you wait a small while then hopefully #979 will be merged in and will remove the need for the DateFormatterService to be autowired in everywhere 😄

Reply:
its tested in `GeneratedOrderEventHandlerTest`, `NoticeOfPlacementOrderUploadedEventHandlerTest`, `CaseManagementOrderRejectedEventHandlerTest` and `CaseManagementOrderIssuedEventHandlerTest`. Reason it was moved into a different config class is to avoid repeating these 2-classes in four places.